documentation/src/gate_operations/intro.md

```diff
@@ -20,14 +20,14 @@ A list of the gate operations available in qoqo and roqoqo with their mathematic
 |---------|---------|
 | Hadamard     | The Hadamard gate, to create a superposition of states, and so to change the basis.  |
 | InvSqrtPauliX     | The inverse square root of the PauliX gate \\( e^{i \frac{\theta}{4} \sigma_x} \\).  |
-| PauliX     | The Pauli X gate, rotation with a fixed angle \\( \frac{\pi}{2} \\) corresponds to a "flip" on x-axis.  |
-| PauliY     | The Pauli Y gate, rotation with a fixed angle \\( \frac{\pi}{2} \\) corresponds to a "flip" on y-axis.  |
-| PauliZ     | The Pauli Z gate, rotation with a fixed angle \\( \frac{\pi}{2} \\) corresponds to a "flip" on z-axis.  |
+| PauliX     | The Pauli X gate, a rotation with a fixed angle of \\( \frac{\pi}{2} \\), corresponds to a "flip" on x-axis.  |
+| PauliY     | The Pauli Y gate, a rotation with a fixed angle of \\( \frac{\pi}{2} \\), corresponds to a "flip" on y-axis.  |
+| PauliZ     | The Pauli Z gate, a rotation with a fixed angle of \\( \frac{\pi}{2} \\), corresponds to a "flip" on z-axis.  |
 | PhaseShiftState0     | Rotation around z-axis by angle \\(\theta\\) applied on state \\( \left \|0 \right> \\) results in a phase shift compared to RotateZ gate. |
 | PhaseShiftState1     | Rotation around z-axis by angle \\(\theta\\) applied on state \\( \left\|1 \right> \\) results in phase shift compared to RotateZ gate. |
 | RotateAroundSphericalAxis     | Implements a rotation around an axis in spherical coordinates.  |
 | RotateX     | The rotation gate around x-axis \\( e^{-i \frac{\theta}{2} \sigma_x} \\).  |
-| RotateXY     | Implements a rotation around an x- and y-axis in spherical coordinates.  |
+| RotateXY     | Implements a rotation around the x- and y-axes in spherical coordinates.  |
```

Comment:
I didn't catch that before, it would be: around an axis in the x-y plane, where the axis is defined by an angle/spherical coordinates